Assets/MixedRealityToolkit.Extensions/Webrtc/Scripts/Marshalling/FrameQueue.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.WebRTC.Marshalling
+{
+    // FramePacket のキュー。
+    // 指定サイズ以上Pushすると、指定サイズ以下になるよう末尾のデータから削除される。
+    // スレッドセーフ。
```

Comment:
Do we want to update the comment?

Reply:
I'll look to you/other MRTK folks for this, as [captured above](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3664#issuecomment-475829055) much of this logic is carried over from another project - not sure if it's best practice to modify this or leave as is.